src/main/java/frc/robot/Config.java

```diff
@@ -194,7 +194,10 @@ public static final class PhotonConfig{
       //-(0.865/2 - 0.095) = 0.3375
     //@todo: new 148 deg camera, measured for Apollo
     public static final Transform3d  leftReefCameraTransform = new Transform3d(
```

Comment:
This camera transform was not correct for the new robot. You will need to remeasure the camera location and update this.